cfgov/templates/wagtailadmin/css/general-enhancements.css

```diff
@@ -78,3 +78,11 @@ li.sequence-member .sequence-member-inner .sequence-type-list .sequence-member-i
 .sequence-member:first-child > .sequence-member-inner > .field > .field-content {
     padding-top: 1em;
 }
+
+/* Fix alignment of multiple choice field labels */
```

Comment:
If you wouldn't mind making this comment a bit more detailed, including mention of the field in TDP that raised a need for it, that'll help future us know what this was specifically for. Rebase on `master` while you're at it, please.

Reply:
Just updated that comment.

Reply:
Thanks!